docs/pages/tutorial/gestures.mdx

```diff
@@ -277,31 +258,23 @@ These translation values will move the sticker around the screen. Since the stic
 In the `useSharedValue()` hooks, we have set both translation variables to have an initial position of `0`.
 This means that the position the sticker is initially placed is considered the starting point. This value sets the initial position of the sticker when the gesture starts.
 
-In the previous step, we triggered the `onActive()` callback for the tap gesture inside the `useAnimatedGestureHandler()` function.
-Similarly, for the pan gesture, we have to specify two callbacks:
-
-- `onStart()`: when the gesture starts or is at its initial position
-- `onActive()`: when the gesture is active and is moving
+In the previous step, we triggered the `onStart()` callback for the tap gesture chained to the `Gesture.Tap()` method.
+Similarly, for the pan gesture, we have to specify an `onChange()` callback which runs when the gesture is active and is moving.
 
-Create an `onDrag()` method to handle the pan gesture.
+Create a `drag` object to handle the pan gesture.
 
 {/* prettier-ignore */}
 ```jsx EmojiSticker.js
-const onDrag = useAnimatedGestureHandler({
-  onStart: (event, context) => {
-    context.translateX = translateX.value;
-    context.translateY = translateY.value;
-  },
-  onActive: (event, context) => {
-    translateX.value = event.translationX + context.translateX;
-    translateY.value = event.translationY + context.translateY;
-  },
-});
+  const drag = Gesture.Pan()
+    .onChange((event) => {
+      translateX.value += event.changeX;
+      translateY.value += event.changeY;
+    });
 ```
 
-Both the `onStart` and `onActive` methods accept `event` and `context` as parameters. In the `onStart` method, we'll use `context` to store the initial values of `translateX` and `translateY`. In the `onActive` callback, we'll use the `event` to get the current position of the pan gesture and `context` to get the previously stored values of `translateX` and `translateY`.
+The `onChange()` callback accepts `event` as parameter. `changeX` and `changeY` properties hold the change in position since the last event. Use them to update the values stored in `translateX` and `translateY`.
```

Comment:
```suggestion
The `onChange()` callback accepts `event` as a parameter. `changeX` and `changeY` properties hold the change in position since the last event. They are used to update the values stored in `translateX` and `translateY`.
```